packages/clerk-js/src/ui/components/OrganizationSwitcher/utils.ts

```diff
@@ -1,3 +1,5 @@
+import type { ClerkPaginatedResponse } from '@clerk/types';
```

Comment:
❓ Should we add tests for these utils?

Reply:
Yes please

Reply:
I couldn't get this to work with swr, so i decoupled the helpers and tested those instead.